app/routers/api/files.py

```
@@ -198,6 +207,59 @@ def _serialize_file(
     return file_dict
 
 
+def _serialize_remote_cached_file(remote_file: RemoteSharedFileCache, peer: P2PPeer) -> dict:
+    """Serialize a cached remote file into the same shape used by /files UI."""
+    file_mtime = remote_file.file_mtime.isoformat() if remote_file.file_mtime else None
+    now_iso = datetime.utcnow().isoformat()
```

Ruff (0.15.12)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)